packages/dialect-react-ui/components/Chat/screens/Main/pages/ThreadPage/Thread.tsx

```diff
@@ -15,6 +15,7 @@ export default function Thread() {
 
   const [text, setText] = useState<string>('');
   const [error, setError] = useState<ParsedErrorData | null | undefined>();
+  const [youCanWrite, setYouCanWrite] = useState<boolean>(false);
```

Comment:
I see that we have been doing this previously, but I think we shouldn't be describing primitive types in generics, since the default value will help us understand what type it is.

What are your thoughts on this? @zhelezkov also prefers explicitly defining the type. If the majority prefers explicitly describing the type, then I'd freeze it as our convention (we should probable define these sooner or later...). Wdyt?

Reply:
I personally like it being explicit but no strong opinion here. Maybe @zhelezkov has some input?

Reply:
Ok, we can come back to this discussion later on, but let's go with explicit for now then.